tests/cli/test_help_and_usage.py

```diff
@@ -35,6 +35,19 @@ def test_version_on_version(cli):
 
 
 def test_current_help_in_readme(cli):
-    p = cli("--help")
```

Comment:
That was what I wanted to avoid. If later we change the text in the help message we will have to change it in two places. I would move this test into `tests/test_interface.py` or similar and test the output of `print_help()` with https://docs.pytest.org/en/latest/capture.html?highlight=stdout#accessing-captured-output-from-a-test-function instead of using `cli("--help")`.

Reply:
I'll take another look at this. I misunderstood, sorry.